driver/docker-compose.yml

```diff
@@ -11,17 +11,16 @@ services:
   truffle:
```

Comment:
I imagine it makes sense for the docker compose file to stay in the root of the repository. Its imaginable that we want the docker compose file to start up the driver and the price estimator together for example.

Reply:
That's reasonable if/when we want to do that but until then it makes more sense in the driver folder to me.

Reply:
The other argument is that `docker-compose up stablex` no longer works from the root (akin to `cargo run -p driver` for example). To me the Docker compose file is really for orchestrating multiple docker images throughout the repository (kind of how a Cargo workspace aggregates multiple crates).

Reply:
I agree with this now. Moving this file back to the root.